apps/entry/tests/test_schemas.py

```diff
@@ -548,7 +568,7 @@ def test(self):
         EntryAttributeFactory.create(
             entry=entry3_2,
             widget=self.widget_date_range,
-            data={'value': {'startDate': '2020-01-15', 'endDate': '2020-01-10'}},
+            data={'value': {'startDate': '2020-01-15', 'endDate': '2020-01-25'}},
```

Comment:
may be write validation for this?

Reply:
Write validation for entry attributes? We have a separate ticket #854 for that.

Reply:
yes.